docs/apm/traces/tracing-ingest.md

```diff
@@ -0,0 +1,59 @@
+---
+id: tracing-ingest
+title: Use Tracing to Troubleshoot Ingest
+sidebar_label: Use Tracing to Troubleshoot Ingest
+description: Use tracing to to search the audit index for all events that impact ingest.
+ 
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The [Audit Event Index](/docs/manage/security/audit-event-index/) contains event logs in JSON format on account activities, allowing you to monitor and audit changes. You can use the `tracingIngest` source category to search the Audit Event Index for all events that impact [log ingestion](/docs/manage/ingestion-volume/log-ingestion/).
+
+## Example query
+In the following example, events logged for `_index=sumologic_system_events AND _sourceCategory=tracingIngest` indicate some impact on ingest. The example query assumes all event types are events to be informed about.
+```_index=sumologic_system_events AND _sourceCategory=tracingIngest
```

Comment:
Drop `_index=...` to the next line so that first line of code block is empty (three backticks only). The other option is to use syntax indicator here (like sql, json, java, etc.).

Reply:
Done.